README.md

```diff
@@ -73,7 +70,7 @@ TODOs
 
 Contributing as a Developer
 --------
-Please see the Contributing.md file before being any new work. 
+Please see the Contributing.md file before being any new work.
```

Comment:
please do not change the lines for formatting. It makes it very difficult to PR when there are so many changes that do nothing.